### ISSUE TITLE
Bump Nodejs to support M1

### ISSUE THREAD

**Issue by loeffel-io** [googleapis/gapic-generator-typescript#1302]
Please bump the [Nodejs version](https://github.com/googleapis/gapic-generator-typescript/blob/main/repositories.bzl#L5) to 

```
http_archive(
    name = "build_bazel_rules_nodejs",
    sha256 = "5aae76dced38f784b58d9776e4ab12278bc156a9ed2b1d9fcd3e39921dc88fda",
    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.7.1/rules_nodejs-5.7.1.tar.gz"],
)
```

**Comment by alexander-fenster** [comment]
I'm actually working on switching from the old Bazel rules to the modern rules from https://github.com/aspect-build/rules_js and https://github.com/aspect-build/rules_ts. It requires some changes in the repository but will make the whole setup work better.

**Comment by alexander-fenster** [comment]
Fixed in #1307.
